src/audio/copier/copier.c

```diff
@@ -1656,8 +1656,14 @@ static uint64_t copier_get_processed_data(struct comp_dev *dev, uint32_t stream_
 	uint64_t ret = 0;
 
 	if (cd->endpoint_num && cd->bsource_buffer != input) {
-		if (stream_no < cd->endpoint_num)
```

Comment:
wrong prefix in commit title. it should be copier: ...

Reply:
done